app/lib/dfe/reference_data/degrees.rb

```diff
@@ -1,730 +1,669 @@
       )
 
-      TOPICS = DfE::ReferenceData::HardcodedReferenceList.new(
-        {
-          'A' => { 'name' => 'Arts' },
-          'A+PGCE' => { 'name' => 'Arts with intercalated PGCE' },
-          'Ed' => { 'name' => 'Education' },
-          'AEcon' => { 'name' => 'Arts in Economics' },
-          'AArch' => { 'name' => 'Arts in Architecture' },
-          'AO' => { 'name' => 'the Art of Obstetrics' },
-          'Asc' => { 'name' => 'Applied Science' },
-          'Ag' => { 'name' => 'Agriculture' },
-          'Acc' => { 'name' => 'Accountancy' },
-          'Admin' => { 'name' => 'Administration' },
-          'BA' => { 'name' => 'Business Administration' },
-          'BS' => { 'name' => 'Business Studies' },
-          'Ch' => { 'name' => 'Chirurgiae' }, # Apparently, this is Latin for 'surgery'
-          'Chem' => { 'name' => 'Chemistry' },
-          'CL' => { 'name' => 'Civil Law' },
-          'Comm' => { 'name' => 'Commerce' },
-          'CS' => { 'name' => 'Combined Studies' },
-          'D' => { 'name' => 'Divinity' },
-          'DS' => { 'name' => 'Dental Surgery' },
-          'Eng' => { 'name' => 'Engineering' },
-          'Eng/BS' => { 'name' => 'Engineering with Business Studies' },
-          'FA' => { 'name' => 'Fine Art' },
-          'GS' => { 'name' => 'General Studies' },
-          'H' => { 'name' => 'Humanities' },
-          'Hyg' => { 'name' => 'Hygiene' },
-          'L' => { 'name' => 'Law' },
-          'Lib' => { 'name' => 'Librarianship' },
-          'LS' => { 'name' => 'Librarianship and Information Studies' },
-          'Litt' => { 'name' => 'Literature' },
-          'LEcon' => { 'name' => 'Land Economy' },
-          'MedSc' => { 'name' => 'Medical Science' },
-          'M' => { 'name' => 'Medicine' },
-          'Met' => { 'name' => 'Metallurgy' },
-          'Met/Eng' => { 'name' => 'Metallurgy and Engineering' },
-          'Mus' => { 'name' => 'Music' },
-          'N' => { 'name' => 'Nursing' },
-          'Pharm' => { 'name' => 'Pharmacy' },
-          'Phil' => { 'name' => 'Philosophy' },
-          'Phys' => { 'name' => 'Physics' },
-          'Sc' => { 'name' => 'Science' },
-          'ScEcon' => { 'name' => 'Science Economy' },
-          'ScEng' => { 'name' => 'Science and Engineering' },
-          'ScTech' => { 'name' => 'Science and Technology' },
-          'Sc SS' => { 'name' => 'Science in Social Science' },
-          'Sc SPT' => { 'name' => 'Science in Speech Therapy' },
-          'SS' => { 'name' => 'Social Science' },
-          'SSt' => { 'name' => 'Social Studies' },
-          'S' => { 'name' => 'Surgery' },
-          'Tech' => { 'name' => 'Technology' },
-          'Theol' => { 'name' => 'Theology' },
-          'VMed' => { 'name' => 'Veterinary Medicine' },
-          'VMS' => { 'name' => 'Veterinary Medicine and Surgery' },
-          'VSc' => { 'name' => 'Veterinary Science' }
-        }
-      )
-
       TYPES = DfE::ReferenceData::HardcodedReferenceList.new(
```

Comment:
I wonder if we should call these 'titles'? Although "types" is probably also fine. 🤷 

```suggestion
      TITLES = DfE::ReferenceData::HardcodedReferenceList.new(
```

Reply:
Yeah, I think "titles" is a bit too specific here